app/google-services.json

```diff
@@ -0,0 +1,69 @@
+{
```

Comment:
Can you format this file using https://jsonformatter.org/ ?

Reply:
Format returned same version.